azavea/raster-foundry
# Minor frontend cleanup

## Overview

This started out as a card to close #1698 , but when that didn't solve any of the problems we are having, I used the remaining time to dig a bit deeper and cleaned up a few things that I found.

This:
- Switches to using the official `node` Docker image; we were using a 3rd-party image that builds off the official image and adds `yarn`, but the latest version of `yarn` is now included in the official images, so using a 3rd-party image is no longer necessary.
- Fixes the webpack loader test regex for detecting font files, which was causing webpack not to properly load fonts with specifications of the form `url('../font/fontello.woff2?96313270')`
- A number of our frontend files were using DOS line endings; this converts them to Unix line endings.

### Checklist


### Notes

I still wasn't able to find a way to address the issues that @designmatty is having; I opened #1731 to track our progress.

## Testing Instructions

 * Run `yarn install` from `app-frontend`
 * SSH into the VM and `docker-compose -f docker-compose.yml -f docker-compose.test.yml run --rm app-frontend install`. Make sure that a new image gets pulled.
 * Copy the fonts regex from `app-frontend/config/webpack/global.js` into any regex parser and confirm that the following strings are detected:
    * `fontello.woff2`
    * `fontello.woff2?96313270`
    * `fontello.eot?96313270#iefix`

Closes #1698
